app/Console/Commands/Cloud/UsagePingCommand.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Pterodactyl\Console\Commands\Cloud;
+
+use Pterodactyl\Models\Egg;
+use Pterodactyl\Models\Nest;
+use Pterodactyl\Models\Node;
+use Pterodactyl\Models\Pack;
+use Pterodactyl\Models\User;
+use Pterodactyl\Models\Server;
+use Illuminate\Console\Command;
+use Pterodactyl\Models\Subuser;
+use Pterodactyl\Models\Database;
+use Pterodactyl\Models\Location;
+use Pterodactyl\Models\Schedule;
+use Pterodactyl\Models\DatabaseHost;
+use GuzzleHttp\Client as GuzzleClient;
+use GuzzleHttp\Exception\RequestException;
+
+class UsagePingCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'p:cloud:ping';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Sends a usage ping';
+
+    /**
+     * @var \GuzzleHttp\Client
+     */
+    protected $client;
+
+    /**
+     * Create a new command instance.
+     *
+     * @param \GuzzleHttp\Client    $client
+     */
+    public function __construct(GuzzleClient $client)
+    {
+        parent::__construct();
+
+        $this->client = $client;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        try {
+            $this->client->request('POST', 'https://pterodactyl.cloud/api/usageData/panel', [
+                'timeout' => config('pterodactyl.guzzle.timeout'),
+                'connect_timeout' => config('pterodactyl.guzzle.connect_timeout'),
+                'json' => [
+                    'uuid' => config('pterodactyl.cloud.uuid'),
+                    'hostname' => config('app.url'),
+                    'appAuthor' => config('pterodactyl.service.author'),
```

Comment:
I don't think any identifiable information should be sent in this, including the author and URL. It should only be a UUID that is not tied to anything publicly, and numbers.

Reply:
This also includes the theme.

Reply:
I can drop the author/url, what if we sent a custom_theme bool if the theme isn't the default pterodactyl one?

Reply:
That works for me. Its probably not super helpful because most people just edit the pterodactyl one anyways, but a bool is fine.